app/lib/mets_directory_scanner.rb

```diff
@@ -0,0 +1,72 @@
+# frozen_string_literal: true
```

Comment:
For your consideration - would it make more sense to have this be a concern - something like `DirectoryScannable` and include it in one of our classes, either the mets document or the batch process, maybe?